doc/audio.md

```diff
@@ -28,9 +31,12 @@ This uses the [audioplayers](https://github.com/luanpotter/audioplayer) lib, in
 If you want to play indefinitely, just use `loop` function:
 
 ```dart
-    Flame.audio.loop('music.mp3');
+    Flame.audio.loop('music.mp3'); // Or
+    Flame.audio.loopLongAudio('music.mp3');
 ```
 
+The difference between the `play/loop` and `playLongAudio/loopLongAudio` is that `play/loop` makes uses of optimezed features that allow sounds to be looped without gaps between their iterations, and no drop on the game frame rate will happen, you should whenever possible, prefer these methods.  `playLongAudio/loopLongAudio` allows any audio of any length to be played, but they do create frame rate drop, and the looped audio will feature a small gap between iterations.
```

Comment:
`and no drop on the game frame` that's a bold overstatement there! hahaha maybe `and almost no drop`, or `smaller drop`;
frame rate will happen, you should whenever... => frame rate will happen. You should, whenever possible, prefer `play/loop` methods.
ds.  `playLongAudio/loopLongAudio` => double spaces there.
allows any audio of any length  => allows for audios of any length

Reply:
Agree. :)